playbooks/roles/bats/tasks/main.yml

```diff
@@ -21,7 +21,13 @@
     dest: "{{ bats_forklift_dir }}"
     update: "{{ bats_update_forklift }}"
 
+- name: "Bats output directory"
+  file:
+    state: "directory"
+    path: "{{ bats_output_dir }}"
+
 - name: "Run bats"
-  shell: "bats {{ bats_tests }}"
+  shell: "bats --tap {{ item }} > {{ bats_output_dir }}/{{ item }}.tap"
```

Comment:
Forgot this change -- specifically do tap output and spit out to `.tap` file type.